client/me/connected-application-item/index.jsx

```diff
@@ -121,7 +121,7 @@ class ConnectedApplicationItem extends React.Component {
 		return (
 			<div>
 				<h2>
-					{ this.props.translate( 'Access Scope' ) }
+					{ this.props.translate( 'Access scope' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.